pkgs/by-name/fi/fido2-manage/package.nix

```diff
@@ -0,0 +1,138 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  cmake,
+  libcbor,
+  openssl,
+  zlib,
+  gnugrep,
+  gawk,
+  # Linux only
+  pcsclite,
+  udev,
+  # GUI
+  python3,
+  xterm,
+  makeDesktopItem,
+  copyDesktopItems,
+  # Darwin only
+  libuv,
+  libsolv,
+  libcouchbase,
+  darwin,
+}:
+let
+  pythonEnv = python3.withPackages (ps: [ ps.tkinter ]);
+in
+stdenv.mkDerivation rec {
+  pname = "fido2-manage";
+  version = "0-unstable-2024-09-24";
+
+  src = fetchFromGitHub {
+    owner = "token2";
+    repo = "fido2-manage";
+    rev = "6aef9ceccdf7bcc60a9298e51a4db633256925df";
+    hash = "sha256-rXTL6wpdvCifakmxH14wBLbhTptNYNFGEPskpUy3IjA=";
+  };
+
+  nativeBuildInputs =
+    [
+      pkg-config
+      cmake
+    ]
+    ++ lib.optionals stdenv.isLinux [
+      copyDesktopItems
+    ];
+
+  buildInputs =
+    [
+      libcbor
+      openssl
+      zlib
+    ]
+    ++ lib.optionals stdenv.isLinux [
+      xterm
+      udev
+      pcsclite
+    ]
+    ++ lib.optionals stdenv.isDarwin [
+      libuv
+      libsolv
+      libcouchbase
+      gnugrep
```

Comment:
grep as build input? Shouldn't this go into native build inputs?

Reply:
my bad. removed it